.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    ...require("@stoplight/eslint-config/prettier.config"),
```

Comment:
just curious - why is it reexported this way?

Reply:
If you mean why not
```js
module.exports = require("@stoplight/eslint-config/prettier.config")
```
then I don't know. This is how it's everywhere, probably the other one would work too but didn't try. I guess this gives us the opportunity to override some rules later on if needed.

This is prettier's config file, so essentially what we're doing here is telling it to inherit all configuration from our common configuration repo.